website/docs/admin_settings_project_anatomy.md

```diff
@@ -113,23 +119,39 @@ Template `{project[code]}_{asset}_{subset}<_{output}><.{@frame}>.{ext}` can hand
 
 ## Attributes
 
-Project attributes are used as default values for new assets created under project, except `Applications` and `Active project` which are project specific. Values of attributes that are **not** project specific are always used from assets. So if `tools` are not loading as expected it is because the assets have different values.
+Project attributes are used as default values for new *folders* created under a project, except `Applications` which is project specific. Values of attributes that are **not** project specific are always used from *folders*. So if `tools` are not loading as expected it is because the *folders* have different values.
 
 ![anatomy_attributes](assets/settings/anatomy_attributes.png)
 
-**Most of attributes don't need detailed explanation.**
-
 | Attribute | Description |
 | --- | --- |
 | `Applications` | List of applications that can be used in the project. At the moment used only as a possible filter of applications. |
 | `Tools` | List of application tools. This value can be overridden per asset. |
-| `Active project` | Project won't be visible in tools if enabled.<br/> - To revert check `Show Inactive projects` checkbox in project settings. |
 
+## Folder Types
+
+:::info
+Folder and its types is a upcoming feature. Templates are not supporting it yet.
+:::
+
+All hierarchical parents are called **folder**. Each folder is having its type for visual differentiation. Folders are also  accessible via templates as `{folder[type]}`.
```

Comment:
We should probably just link to the folder section in artist key concepts, so we're not explaining the same thing in the glossary and other places too. 

It's also a bit confusing that we're showing their setup, but saying they are upcoming feature. The are already fully functional, ,just don't work in the templates, so I'd simplify here a little bit in general